packages/create-react-app/createReactApp.js

```diff
@@ -124,6 +125,10 @@ const program = new commander.Command(packageJson.name)
   })
   .parse(process.argv);
 
+if (isInstalledGlobally) {
+  console.log(`global installation for create-react-app is deprecated.`);
```

Comment:
I think is good if you use `console.warn` with `chalk.yellow` to give a better warning message:
```
console.warn(`${chalk.yellow('global installation for create-react-app is deprecated.')}`); 
```

Reply:
I don't agree @Taym95, as this is Node, not the browser.

But I do agree that we should use colouring! :)

A few other thoughts:
- Perhaps this should be the first thing emitted from the script? Above `program`?
- Could the copy be more descriptive? Something like:

> Global installations of `create-react-app` are now deprecated.
>
> To remove the globally installed package, run `npm uninstall -g create-react-app`.
>
> Updated installation instructions can be found here: https://facebook.github.io/create-react-app/docs/getting-started#quick-start

Reply:
We may need a short URL for that - @ianschmitz, do you know if we have one?

Reply:
Actually @Taym95, I've noted that we do use `console.warn` in other places in our Node scripts - so that's a great suggestion ;)

Reply:
Not to my knowledge @mrmckeb.